test/mocha/browser/browser.spec.js

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { pick } from '../lib/object';
+
+const diagramXML = require('./diagram.bpmn');
+
+console.log(diagramXML);
```

Comment:
```suggestion
```

Reply:
Or I might not understand how the tests work and this test that console.log is allowed?

Reply:
I added this on purpose to not trigger `no-unused-var`. The snippets in the tests are not executed but linted only.

Reply:
I was to quick to post this review, makes sense to have this one here 👍

Reply:
Thank you for clarifying